README.md

```diff
@@ -102,6 +102,26 @@ Your project will be compiled ahead of time (AOT), and then the resulting bundle
 - UMD modules result in code that cannot be properly optimized. For best results, prefer ES6 modules whenever possible. This includes third-party dependencies: if one is published in both UMD and ES6 modules, go with the ES6 modules version.
 - During a production build, CommonJs modules will be automatically converted to ES6 modules. This means you can use them and/or require dependencies that use them without any issues.
 
+# Internationalization
+
+Put `i18n` attribute to your html tag to mark it for translation, more information here: https://angular.io/docs/ts/latest/cookbook/i18n.html
+
+## Create a translation source file
+
+```bash
+# Your translation file will be generated here `src/client/assets/locale`
```

Comment:
Could this be made configurable?
I would personally prefer not to have generated files in the `src`.

Also depending on your translation process, it would be good to keep the source and translated files separate.